src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Views/EmployerRequest/Active.cshtml

```diff
@@ -50,7 +50,7 @@
                                         <p class="govuk-body request-item"><b>Employers:</b> @aggregatedEmployerRequest.NumberOfEmployers</p>
                                     </div>
                                     <div class="govuk-grid-column-one-third govuk-body govuk-!-text-align-right--from-tablet">
-                                        <a class="govuk-link govuk-link--no-visited-state govuk-body" asp-route="@EmployerRequestController.SelectRequestsToContactRouteGet" asp-all-route-data="requestRouteData">View requests<span class="govuk-visually-hidden">looking for requests for @aggregatedEmployerRequest.StandardTitle</span></a>
+                                        <a class="govuk-link govuk-link--no-visited-state govuk-body" asp-route="@EmployerRequestController.SelectRequestsToContactRouteGet" asp-all-route-data="requestRouteData">View requests<span class="govuk-visually-hidden">View requests for @aggregatedEmployerRequest.StandardTitle</span></a>
```

Comment:
I would question this as I think this would be read aloud as "View requests View requests for Bricklayer" and I would of thought it should read "View requests for Bricklayer"

With the hidden bit just being " for bricklayer"

Reply:
fixed